apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -165,6 +174,39 @@ export function generateReactHooks<
         }, [rowAction, onComplete])
     }
 
+    function useCount (variables: QueryVariables, options?: QueryHookOptions<IUseCountQueryReturnType, QueryVariables>) {
```

Comment:
Is it able to avoid code duplication? Looks like `useCount` should wrap `useObjects` and return only what is necessary.